blazor/file-manager/upload.md

```diff
@@ -0,0 +1,175 @@
+
+This property lets you define which file types can be uploaded by specifying allowed extensions, separated by commas. For example, to allow only image files, you would set the `AllowedExtensions` property to .jpg,.png.
+
+By setting the `AllowedExtensions` property, you restrict the file types that can be uploaded. Only files with the specified extensions will be accepted.
+
+If you want to allow only image files like .jpg and .png, you would set the property as follows:
+
+```cshtml
+
+@using Syncfusion.Blazor.FileManager
+
+<SfFileManager TValue="FileManagerDirectoryContent">
+    <FileManagerUploadSettings  AllowedExtensions=".jpg,.png"></FileManagerUploadSettings>
+    <FileManagerAjaxSettings Url="https://ej2-aspcore-service.azurewebsites.net/api/FileManager/FileOperations"
+                             UploadUrl="https://ej2-aspcore-service.azurewebsites.net/api/FileManager/Upload"
+                             DownloadUrl="https://ej2-aspcore-service.azurewebsites.net/api//FileManager/Download"
+                             GetImageUrl="https://ej2-aspcore-service.azurewebsites.net/api/test/FileManager/GetImage">
+    </FileManagerAjaxSettings>
+</SfFileManager>
+
+```
+![Blazor FileManager with AllowedExtensions](images/blazor-filemanager-allowextension.png)
+
+## Upload Mode
+
+The [UploadMode](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.FileManager.FileManagerUploadSettings.html#Syncfusion_Blazor_FileManager_FileManagerUploadSettings_UploadMode) property defines the method used to perform the upload operation in the FileManager component.
+
+This property lets you choose between two upload modes. `FormSubmit` Uses the traditional form submission method for file uploads.
+`HttpClient` Uses the HttpClient instance for the upload, providing more control over the request.
+
+By default, the `UploadMode` is set to `FormSubmit`, but you can switch to HttpClient for more control, such as managing headers or authorizing the upload response.
+
+```cshtml
+
+@using Syncfusion.Blazor.FileManager
+
+<SfFileManager TValue="FileManagerDirectoryContent">
+    <FileManagerUploadSettings  UploadMode="UploadMode.FormSubmit"></FileManagerUploadSettings>
+    <FileManagerAjaxSettings Url="https://ej2-aspcore-service.azurewebsites.net/api/FileManager/FileOperations"
+                             UploadUrl="https://ej2-aspcore-service.azurewebsites.net/api/FileManager/Upload"
+                             DownloadUrl="https://ej2-aspcore-service.azurewebsites.net/api//FileManager/Download"
+                             GetImageUrl="https://ej2-aspcore-service.azurewebsites.net/api/test/FileManager/GetImage">
+    </FileManagerAjaxSettings>
+</SfFileManager>
+
+```
+![Blazor FileManager with UploadMode](images/blazor-filemanager-uploadmode.png)
```

Comment:
Remove the image and modify the code to authorize upload response @saravananj-sf4375

Reply:
modified the code and updated